deploy/iso/minikube-iso/package/podman/podman.mk

```diff
@@ -1,4 +1,4 @@
-PODMAN_VERSION = v1.0.0
+PODMAN_VERSION = v1.2.0
```

Comment:
deploy/iso/minikube-iso/package/podman/podman.hash will also need to be updated.

You can verify that the build works by typing `make out/minikube.iso`: https://github.com/kubernetes/minikube/blob/master/docs/contributors/minikube_iso.md

Reply:
You also need to update the git commit, it is used by the versioning system.

See https://github.com/containers/libpod/releases

Most likely, this upgrade needs to be timed with the system `runc` upgrade.

The plan is to use the _same_ binary for docker/podman/cri-o, hope it works...